tests/simpleTest/persistency_test.cpp

```diff
@@ -90,6 +90,7 @@ TEST_F(PersistencyTest, PrimaryRestartVC) {
     ISimpleTestReplicaBehavior *b = new OneTimePrimaryDownVC(rp);
     create_and_run_replica(rp, b);
   }
+  // abort();
```

Comment:
Can we remove this line? I assume it was just for testing failure.

Reply:
yes, it was for testing. removed